scrapper.js

```diff
@@ -0,0 +1,85 @@
+var express = require('express')
+var request = require('request')
+var cheerio = require('cheerio')
+var app     = express()
+
+app.get('/scrape', function(req, res){
+    
+  var url = 'https://playoverwatch.com/en-us/career/pc/us/' + req.query.name
```

Comment:
Make the string part of the url a constants. See `constants/` folder for examples.